src/main/java/org/terasology/weatherManager/components/WeatherBase.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
You may want to use the shorter license headers we introduced some time back.

```suggestion
// Copyright 2020 The Terasology Foundation
// SPDX-License-Identifier: Apache-2.0
```

There should be an IDE setting somewhere which is supposed to be automatically picked up when loading via `build.gradle`.